plugins/node/opentelemetry-instrumentation-express/examples/package.json

```diff
@@ -32,16 +33,18 @@
     "@opentelemetry/exporter-jaeger": "^0.25.0",
     "@opentelemetry/exporter-zipkin": "^0.25.0",
     "@opentelemetry/instrumentation": "^0.25.0",
-    "@opentelemetry/instrumentation-express": "^0.23.0",
     "@opentelemetry/instrumentation-http": "^0.25.0",
     "@opentelemetry/resources": "^0.25.0",
-    "@opentelemetry/semantic-conventions": "^0.25.0",
-    "@opentelemetry/sdk-trace-node": "^0.25.0",
     "@opentelemetry/sdk-trace-base": "^0.25.0",
+    "@opentelemetry/sdk-trace-node": "^0.25.0",
```

Comment:
Does the move really result in up to date samples if they don't use up to date dependencies? I think they should use the current instrumenation and the current core dependencies.

Reply:
Good catch! I agree. Do you think some of them should be referenced with relative paths, version `*`, or something else?

Reply:
well, that's one of the point where it's easy to complain but hard to provide a good solution.
I think we should aim to depend on the same versions then the instrumentation does for that ones which are in common.
The sample should most likely depend on the instrumentation in the repo, maybe link via lerna or similar.
And renovate bot should include the samples.

But I don't know if this is easy to archive.

Reply:
If you add the express instrumentation to the package.json as a relative path it might be good enough. I would then use `*` dependencies for otel dependencies so we always get latest (or `^1.0.3` if you prefer).